locations/spiders/whole_foods.py

```diff
@@ -0,0 +1,71 @@
+# -*- coding: utf-8 -*-
+import scrapy
+from locations.items import GeojsonPointItem
+import re
+import string
+
+class WholeFoodsSpider(scrapy.Spider):
+    name = "whole_foods"
+    allowed_domains = ["www.wholefoodsmarket.com"]
+    start_urls = (
+        'https://www.wholefoodsmarket.com/stores/list?page=0',
```

Comment:
Looks like you can get lat/lon by starting at `https://www.wholefoodsmarket.com/ajax/stores`, then getting more info by spidering to "https://www.wholefoodsmarket.com/stores/<path>" where `<path>` is in the JSON response.

Reply:
I'll rewrite the spider tomorrow using this